config/catkin_lint.upstream.yaml

```
@@ -1,6 +1,6 @@
 name: catkin_lint
 method: http://ppa.launchpad.net/roehling/ros/ubuntu
-suites: [xenial, bionic, focal]
+suites: [bionic, focal]
 component: main
 architectures: [i386, amd64, armhf, arm64, source]
-filter_formula: Source (== catkin-lint), $Version (% 1.6.12-*)
+filter_formula: $Source (= catkin-lint), $Version (% 1.6.14-*)
```

```suggestion
filter_formula: (Package (= catkin-lint), $Source (= catkin-lint)), $Version (% 1.6.14-*)
```

aptly which is currently in use on the bootstrap repository rather than reprepro doesn't recognize source packages as matching their own $Source so in order to import the catkin-lint source package we also need to match on the package name `catkin-lint`.

We didn't have checks for source package availability previously which is why this didn't come up before.

---

Whoops. Sorry about e2c9ce3. I didn't transcribe it properly from my local test environment.